spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/logging/NacosLoggingListener.java

```diff
@@ -30,6 +30,7 @@
  *
  * @author mai.jh
  */
+@Deprecated
```

Comment:
Whether we delete the invalid listener directly?

Reply:
The reason why these two classes were not directly deleted is that there may be users who depend on these classes for some customization operations (although this possibility is small), so they are marked as `deprecated` first and will be removed in the next version.